pkg/controller/deployment/deployment_controller.go

```diff
@@ -67,27 +67,27 @@ type DeploymentController struct {
 	syncHandler func(dKey string) error
 
 	// A store of deployments, populated by the dController
-	dStore cache.StoreToDeploymentLister
+	dLister cache.StoreToDeploymentLister
 	// Watches changes to all deployments
 	dController *cache.Controller
 	// A store of ReplicaSets, populated by the rsController
-	rsStore cache.StoreToReplicaSetLister
+	rsLister cache.StoreToReplicaSetLister
```

Comment:
I assume the rs changes in this commit should really be in the rs commit, but I'm not going to make you change them 😄